src/test/java/JPPParserTest.java

```diff
@@ -32,37 +36,40 @@ private record TestCase<Input, Expected>(Input input, Expected expected) {
     private record ThrowingTestCase(String formula) {
     }
 
-    private static List<JPPParserTest.TestCase<String, String>> abstractionTests() {
+    private static List<JPPParserTest.TestCase<String, Node>> abstractionTests() {
         return List.of(
                 /// #if expression
                 // expression := <operand> <operator> <operand> | [!]defined(name)
                 // expression := operand == operand
-                new JPPParserTest.TestCase<>("//#if 1 == -42", "1__EQ____U_MINUS__42"),
+                new JPPParserTest.TestCase<>("//#if 1 == -42", new Literal("1==-42")),
                 // expression := operand != operand
-                new JPPParserTest.TestCase<>("// #if 1 != 0", "1__NEQ__0"),
+                new JPPParserTest.TestCase<>("// #if 1 != 0", new Literal("1!=0")),
                 // expression := operand <= operand
-                new JPPParserTest.TestCase<>("//#if -1 <= 0", "__U_MINUS__1__LEQ__0"),
+                new JPPParserTest.TestCase<>("//#if -1 <= 0", new Literal("-1<=0")),
                 // expression := operand < operand
-                new JPPParserTest.TestCase<>("//#if \"str\" < 0", "__QUOTE__str__QUOTE____LT__0"),
+                new JPPParserTest.TestCase<>("//#if \"str\" < 0", new Literal("\"str\"<0")),
                 // expression := operand >= operand
-                new JPPParserTest.TestCase<>("//   #if \"str\" >= \"str\"", "__QUOTE__str__QUOTE____GEQ____QUOTE__str__QUOTE__"),
+                new JPPParserTest.TestCase<>("//   #if \"str\" >= \"str\"", new Literal("\"str\">=\"str\"")),
                 // expression := operand > operand
-                new JPPParserTest.TestCase<>("//  #if 1.2 > 0", "1__DOT__2__GT__0"),
+                new JPPParserTest.TestCase<>("//  #if 1.2 > 0", new Literal("1.2>0")),
                 // expression := defined(name)
-                new JPPParserTest.TestCase<>("//#if defined(property)", "DEFINED_property"),
+                new JPPParserTest.TestCase<>("//#if defined(property)", new Literal("defined(property)")),
                 // expression := !defined(name)
-                new JPPParserTest.TestCase<>("//#if !defined(property)", "__U_NOT__DEFINED_property"),
+                new JPPParserTest.TestCase<>("//#if !defined(property)", new Literal("defined(property)", false)),
```

Comment:
Maybe it would be a good idea to use and extend our `FormulaUtils` (`import static org.variantsync.diffdetective.util.fide.FormulaUtils.*;`) for these tests. That class defines functions for correction negation of formulas as well as some nice wrappers for the constructors so we do not have to write `new` all the time (e.g., `and(...)` instead of `new And(...)`).

Reply:
As an example, you might want to have a look at `ViewTest`. This is inconsistent across all tests so it might better be done in a new PR.